### DISCUSSION TITLE
How can I change the axis limit value to be able to use the classic Wii controller?

### DISCUSSION THREAD

**Discussion by thehanker01** [mupen64plus-ae/mupen64plus-ae#1044] [Q&A]
Hello, first thank you very much for the emulator, it is very good. Now the question:
How can I change the axis limit value to be able to use the classic Wii controller? Currently the value of the right stick is 0.40 and Mupen does not recognize it. It seems to me that from 0.50 it recognizes it, is it possible to go down to 0.35 or 0.40?

**Comment by fzurita** [comment]
I'm sorry, I'm not sure what you are referring to exactly when it comes to "axis limit".

Can you include screenshots of the app to see what you are referring to?

**Reply by thehanker01** [reply]
Sure, I'll upload the screenshots as soon as I can.
I'll explain, when connecting a classic control to the wiimote and configuring the stick to move the character is very slow, and I have to increase the sensitivity to 200%, and that's how it works without problems (left stick) and on the right stick I map the C buttons , but the emulator does not recognize them, because they are very insensitive.

I hope you understand, because I use a translator and I speak Spanish.

I'll send a screenshot anyway.

Thanks / Gracias

**Comment by fzurita** [comment]
Gotcha, I do understand. The dead zone is preventing then analog stick to be registered as C buttons.

Have you tried decreasing the dead zone?

That may not work though, I need to remind myself of that area of the code.

**Comment by thehanker01** [comment]
I will try to reduce the dead zone, but I think I will try it without result, increasing it to the maximum (40) or decreasing it to zero. But I will try again what you tell me.

**Comment by thehanker01** [comment]
Here are the screenshots:

Wiimote with Classic Control

Right Stick: C buttons no working

![Screenshot_20220721-145459_M64Plus_FZ_Pro.png](https://user-images.githubusercontent.com/109752193/180298643-dd816973-fd1b-4f11-9c15-cfd90d8c2426.png)

Left Stick: C buttons working (in map left stick)

![Screenshot_20220721-145521_M64Plus_FZ_Pro.png](https://user-images.githubusercontent.com/109752193/180298823-67a3644b-da69-4b68-8877-3a6ad03fcc9a.png)

Other Bluetooth Controller:

Left and Right Stick

![Screenshot_20220721-145812_M64Plus_FZ_Pro.png](https://user-images.githubusercontent.com/109752193/180298992-2c25736c-3c88-4723-8ec3-089e57b4e32f.png)

![Screenshot_20220721-145840_M64Plus_FZ_Pro.png](https://user-images.githubusercontent.com/109752193/180299014-1297bd14-073c-46f2-8dfd-4b324185f7c8.png)

**Comment by fzurita** [comment]
Looks like I just need to have the dead zone apply for determining when a button is pressed as well. We are not doing that right now, so it's essentially a bug.

**Comment by thehanker01** [comment]
Ok, thanks

**Comment by thehanker01** [comment]
where do I find the file to configure the controls manually?

**Reply by fzurita** [reply]
You can export it by going to tools then "import/export data" then "export cheats and profiles". You can then modify it and "import cheats and profiles" for the app to load your modifications.

That won't fix this problem. I need to fix the code unfortunately.
